packages/lib/src/components/BacsDD/components/BacsInput.scss

```diff
@@ -0,0 +1,44 @@
+@import '../../../style';
+.adyen-checkout__bacs{
+
+  &--confirm{
+    position: relative;
+
+    .adyen-checkout-input__inline-validation--valid{
+      display: none
+    }
+  }
+
+  .adyen-checkout__field{
+
+    &--inactive{
+      pointer-events: none;
+    }
+  }
+
+  .adyen-checkout__bacs--edit{
+
+    position: absolute;
+    top: -25px;
+    right: 0px;
+    cursor: pointer;
+    width: 20%;
+
+    &-dropin{
+      top: -50px;
+    }
+
+    .adyen-checkout__bacs--edit-button{
+      border: none;
+      background: none;
+      color: blue;
```

Comment:
We could use here one of the colors in `colors.scss`

Reply:
done